docs/migrations/ibc-migration-042.md

```diff
@@ -0,0 +1,54 @@
+# Migrating to ibc-go
+
+This file contains information on how to migrate from the IBC module contained in the SDK 0.41.x line to the IBC module in the ibc-go repository based on the 0.42 SDK version. 
```

Comment:
I think we also need to add the migration command that's run on `x/genutil/legacy` to the ibc-go repo. So that a migration on gaia uses the `Migration` funcs from both the SDK and ibc-go

Reply:
I see, this migration function is primarily for migrating genesis though? Is there anything we need to do in that migration for this upgrade? Opened: #11